pkg/cmd/datamgr/cli/install/install.go

```diff
@@ -249,14 +249,14 @@ func (o *InstallOptions) CheckPluginImageRepo(f client.Factory) error {
 	tag := ""
 	for _, container := range deployment.Spec.Template.Spec.InitContainers {
 		if strings.Contains(container.Image, utils.VeleroPluginForVsphere) {
-			repo = strings.Split(container.Image, "/")[0]
+			repo = utils.GetRepo(container.Image)
```

Comment:
search util function something like `strings.SplitN`, which could help you find the last `/`.

Reply:
Changed to use strings.LastIndex to find the last '/'.

Reply:
Let's add the image string into the error (line 262).  If this fails, we will not get any indication of what the image name was.